main/speech/TTS.py

```diff
@@ -65,7 +66,7 @@ def speak_google_tts(text):
     """
     with tempfile.TemporaryDirectory() as tmpdirname:
         fd, mpiii = tempfile.mkstemp(suffix='.mp3', dir=tmpdirname)
-        Speech(text=text, lang='en').save(mpiii)
+        Speech(text=text, lang=susi_config["language"]).save(mpiii)
```

Comment:
@norbusan , this only changes speech for google TTS, what about the other two ?(Watson and Flite)

Reply:
@stealthanthrax Watson: I don't know whether/how this can be implemented.
FLite: we would need to download more voice files, and maybe they are not present. Not sure what to do...

Reply:
I updated the branch to select a voice according to the language setting from the list of supported watson voices https://cloud.ibm.com/apidocs/text-to-speech

For flite we need to think ...